lib/net/ssh/transport/session.rb

```diff
@@ -61,12 +61,12 @@ def initialize(host, options={})
       @bind_address = options[:bind_address] || nil
       @options = options
 
-      debug { "establishing connection to #{@host}:#{@port}" }
-
       @socket =
         if (factory = options[:proxy])
+          debug { "establishing connection to proxy" }
```

Comment:
Nope, it doesn't make sense to place `@host` or `@port` because the proxy config ignores those settings entirely. 

If I have an SSH config that says "when I connect to foo.bar.com, proxy me baz.quux.org" then this line would state "connecting to foo.bar.com" even though there is no connection actually being made to "foo.bar.com."